notebooks/azureml_model_headers_demo/german_credit_azure_ml_demo.ipynb

```diff
@@ -1048,12 +736,12 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 36,
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
     "# create an evaluation object and pass the evaluation dataset(csv) here \n",
-    "german_credit_eval_data_file = \"data/german_credit_eval.csv\"\n",
+    "german_credit_eval_data_file = \"../utils/german_credit_eval.csv\"\n",
```

Comment:
This looks suspicious.  Are you sure this shouldn't be `../datasets/german_credit_eval.csv` ??